tools/CoqMakefile.in

```diff
@@ -867,6 +867,11 @@ debug:
 
 .DEFAULT_GOAL := all
 
+# This file can be created by the user to hook into double colon rules
+# or add any other Makefile code he may need, making use of defined
+# variables if necessary.
```

Comment:
```suggestion
# Users can create LOCAL_LATE_FILE to hook into double colon rules
# or add other needed Makefile code, using defined
# variables if necessary.
```

Reply:
You need `@` so that the script does substitution appropriately.
```suggestion
# Users can create @LOCAL_LATE_FILE@ to hook into double-colon rules
# or add other needed Makefile code, using defined
# variables if necessary.
```